tests/basic_setup/mkdocs_w_contribution.yml

```diff
@@ -6,4 +6,5 @@ plugins:
     - git-authors:
         count_empty_lines: false
         show_contribution: true
-        show_line_count: true
\ No newline at end of file
+        show_line_count: true
+        authorship_threshold_percent: 50
```

Comment:
Same comments: 

- This might break existing unit tests. 
- Better to create a new, separate `mkdocs_w_author_threshold.yml`.

I'd also like to have a unit test for author threshold. Are you able to do this, or shall I pick it up?